.gitignore

```diff
@@ -1,6 +1,5 @@
 .directory
 __pycache__
 .cache
-
-library_coverage_xml_and_fulltext_indicators.db*
+./library_coverage_xml_and_fulltext_indicators.db*
```

Comment:
This seems like it would ignore `data/library_coverage_xml_and_fulltext_indicators.db.xz` but its currently tracked?

Reply:
`data/library_coverage_xml_and_fulltext_indicators.db.xz` is tracked because it's listed in the [`.gitattributes`](https://github.com/greenelab/library-access/pull/12/files#diff-fc723d30b02a4cca7a534518111c1a66) file, for use with Git LFS, as I remember/understand.

Reply:
`.gitignore` takes precedent over `.gitattributes`. Probably what happened is you tracked it before adding it to ignore. In the next PR, you will probably have to change this line to remove `*`, so it only ignore the uncompressed.